Vizzuality/gfw
# Feature/gfw overflow menu on user areas

## Overview

Add pop-up message to the MyGFW page letting users know that they can access dashboards for their area by clicking on the thumbnail image

This includes changes in 3 places: 

1. map -> AREAS OF INTEREST new overflow menu
2. area dashboard new overflow menu 
3. myGFW tooltip to indicate that you can open dashboard when clicking on an saved area.

## Demo

--

## Notes

Notion: https://www.notion.so/9dd860f5dfd64475b92f4a3f79f0ff37?v=2098a476c4fa407981718cb4eafad507&p=5808bedc1fb844c0935e8e647496ed3c

## Testing

- Make sure all links go to dashboard, edit area, share area works as expected. from map -> mygfw -> areas menu
- MyGFW page, should show tooltip **Open dashboard** when interacting with it
- Make sure active dashboard has new overflow menu, containing download data, open map, edit area, and if location type is country save_area. Make sure these buttons perform the correct actions when pressed.
